e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -0,0 +1,128 @@
+import TextOptions from './TextOptions';
```

Comment:
Most of these functions seem very boilerplate-y, and seem like things that could live in any project. I believe I made this comment in the past but I don't remember the resolution. Can you add a comment here or maybe a README that explains why this layer is necessary in our application, and what goes here?

Reply:
reworked `xpath-defaults`. almost no boilplate duplicate code.